.pre-commit-config.yaml

```diff
@@ -0,0 +1,14 @@
+repos:
+- repo: https://github.com/antonbabenko/pre-commit-terraform
+  rev: v1.76.0 # Get the latest from: https://github.com/antonbabenko/pre-commit-terraform/releases
+  hooks:
```

Comment:
should we make files explicit for these hooks like for `cloudformation`?
e.g. for each hook:
```
files: terraform/.*\.(tf|hcl)$
```
Without scope defined they will run for each sub-directory, won't they?

Reply:
I guess each hook already has a defined set of files that are whitelisted. You can check them here
https://github.com/antonbabenko/pre-commit-terraform/blob/master/.pre-commit-hooks.yaml
If you feel we should be more restrictive (and less flexible) we can also include the folder in all hooks configurations

Reply:
You are right, let's make some bench testing when we have more files in the repository :+1: